src/components/typography/typography.stories.mdx

```diff
@@ -126,6 +126,79 @@ It's also possible to create ordered and unordered lists using the `List`, `List
   </Story>
 </Preview>
 
+## Examples of white-space
+The `whiteSpace` prop sets how white space inside of an element is handled. The `whiteSpace` prop accepts any valid 
+CSS string.
+
+<Preview>
+  <Story name="white-space" parameters={{ info: { disable: true } }}>
+    {() =>
+      ["normal", "nowrap", "pre", "pre-wrap", "pre-line", "break-spaces"].map((variant) => (
+        <div style={{padding: "32px"}}>
+          <div style={{height: "150px", width: "150px", backgroundColor: "yellow"}}>
+            <Typography whiteSpace={variant}>
+              The is an example of using white-space. This is an example of some text with {variant} applied.
+            </Typography>
+          </div>
+        </div>
+      ))
+    }
+  </Story>
+</Preview>
+
+## Examples of word-wrap
+The `wordWrap` prop allows longer words to be broken and wrapped on to the next line. The `wordWrap` prop
```

Comment:
I'm not sure if there's any benefit from showing these stories as we don't have stories for other css rules